### DISCUSSION TITLE
Different weather providers

### DISCUSSION THREAD

**Discussion by itsHenry35** [valinet/ExplorerPatcher#1482] [Ideas]
As Google is blocked in China, I'd appreciate you if you could add other providers such as qweather.com,  accuweather and so on to use as an alternative weather data source. Quering the API will be a way to unify the UI and so on. Perhaps there could also be an option to let the users enter their own api keys.

**Comment by pyrates999** [comment]
I'm pretty sure the weather feature is a feature in windows.  So it is controlled by Microsoft, not explorer patcher.  And thus is not something explorer patcher can do.
